src/parser/image.ts

```diff
@@ -902,6 +902,13 @@ class DCMImage implements IImageInfo {
 		return this.getTagValueIndexed(TagIds.TransferSyntax) as TransferSyntax;
 	}
 
+	/**
+	 * Sets the tranfer syntax - so we can override it
+	 */
+	set transferSyntax(syntax: TransferSyntax) {
+		this.getTag(TagIds.TransferSyntax).convertedValue = syntax;
```

Comment:
Does this need validating? What happens if I call transferSyntax = "JunkSyntax" from a JS file (not TS, so no type validation)?